apps/api/src/app.module.ts

```diff
@@ -28,7 +29,22 @@ const coreModules = [
 ];
 
 @Module({
-  imports: [...coreModules, ...apiModules],
+  imports: [
+    ...coreModules,
+    ...apiModules,
+    AuthModule.forRoot({
+      connectionURI: process.env.SUPERTOKENS_URI as string,
+      apiKey: process.env.SUPERTOKENS_API_KEY,
+      appInfo: {
+        appName: 'platform',
+        apiDomain: process.env.SUPERTOKENS_API_DOMAIN as string,
+        websiteDomain: process.env.SUPERTOKENS_WEBSITE_DOMAIN as string,
+        apiBasePath: '/auth',
```

Comment:
Change all these into env with default value as /auth

Change app name to process.env